.github/workflows/cicd.yml

```diff
@@ -0,0 +1,44 @@
+name: Create and publish a Docker image
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: ['master']
+  pull_request:
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.TEMP_GHCR_PAT }}
```

Comment:
@matthewshirley I know it's not set on this repo yet, but is this here what I should be using to push to our ghcr?

Reply:
You might want an organization secret
https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-an-organization

Reply:
Sounds good @ryanluker .
Are you able to create one and add the name here as a comment please?

Reply:
`TEMP_GHCR_PAT` was my personal access token that was used for prototyping. Organization secrets are the way to go.

Reply:
@ryanluker I see we have a `READ_PACKAGES_PAT` org secret, but this job will need a `WRITE...` secret.

Reply:
@jeffothy The documentation indicates you should now be using the `GITHUB_TOKEN` to authenticate: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry

```suggestion
          password: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Even better, no need to create an org secret then!

Reply:
@jeffothy I fixed it here https://github.com/grnspace/docker-ngrok/pull/1/commits/c99743b605acda8a55aa828a46dcff4306f2558f